analysis/filter/StructureModifier/build.gradle

```diff
@@ -7,10 +7,13 @@ dependencies {
     testImplementation group: 'org.jetbrains.kotlin', name: 'kotlin-test', version: kotlin_version
     testImplementation project(path: ':model', configuration: 'testOutput')
 
-    testImplementation group: 'junit', name: 'junit', version: junit4_version
+    testImplementation('org.junit.jupiter:junit-jupiter:5.6.0')
```

Comment:
Please keep the former notation (it is also possible to use a variable for the version) and move the version out in a variable (if there's none already that could be used instead)